packages/fxa-payments-server/src/store/actions/index.ts

```diff
@@ -29,4 +29,23 @@ export type ActionFunctions = {
   [key in ActionsKey]: FunctionWithIgnoredReturn<ActionsCollection[key]>;
 };
 
+type PromisePayloadSequenceCreator = (
```

Comment:
This looks like a tangly bit of TypeScript. The purpose is to figure out the eventual return type of an action, unwrapping it from sequences & promises. This is intended as plumbing that should help convert more `any` types from API calls into concrete types.